.github/workflows/instrumentations_0.yml

```diff
@@ -76,8 +76,6 @@ jobs:
         exclude:
           - python-version: py312
             package: "boto"
-          - python-version: py312
-            package: "kafka-python"
```

Comment:
So while kafka-python does not currently support py312, kafka-python-ng does and I removed this.

Reply:
A bit confused on how this could possibly work, I think we should rename the rule and and a specific entry here in the workflow for the `-ng` package, see https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2537/files#r1699836631

Reply:
In that case, I'd vote to keep as-is and just add `kafka-pythonng` to CI

Reply:
Since we are separating out kafka-python-ng and kafka-python, i readded this and followed fastapislim's example in instrumentations_1.yaml